model/Dataset/Properties/intendedUse.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# intendedUse
+
+## Summary
+
+The field describes what the given dataset should be used for.
+
+## Description
+
+Some datasets are collected to be used only for particular purposes. For example, medical data collected from a specific demography might only be applicable for training machine learning models to make predictions for that demography. In such a case, the intendedUse field would capture this information. Similarly, if a dataset is collected for building a facial recognition model, the intendedUse field would specify that.
```

Comment:
Is this field meant to capture only the _intended_ use (e.g. "I think you should only use the dataset for X"), or to capture _restrictions_ on use (e.g. "You may not use the dataset for Y")?

If it's the latter, I think that would more appropriately be handled by representing a license with that restriction (most likely via a `LicenseRef-` license), rather than having a separate data field for what is essentially a licensing restriction somewhere outside the licensing profile.

Reply:
With AI applications, builders typically 'document' their intended use, but it's typically made in the context of positive so not use a restricted license profile would work.  Hope that makes sense on why license information wasn't used.

Reply:
That makes sense -- thank you @bennetkl!